src/bun.js/webcore/response.zig

```diff
@@ -2015,14 +2028,22 @@ pub const Fetch = struct {
 
                         if (options.get(ctx, "tls")) |tls| {
                             if (!tls.isEmptyOrUndefinedOrNull() and tls.isObject()) {
+                                if (SSLConfig.inJS(globalThis, tls, exception)) |config| {
+                                    if (ssl_config) |existing_conf| {
```

Comment:
is there anycase that this would not be null?

Reply:
Nope you're right. I'll remove this